ab2d-contracts-client/src/main/java/gov/cms/ab2d/contracts/model/ContractAPI.java

```diff
@@ -0,0 +1,23 @@
+package gov.cms.ab2d.contracts.model;
+
+import gov.cms.ab2d.contracts.model.Contract;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+
+public interface ContractAPI {
+    @GetMapping("/contract")
+    List<ContractDTO> getContracts(@RequestParam("contractId") Optional<Long> contractId);
```

Comment:
No, it's more like this: @RequestParam(required = false) Long contractId. Also, didn't we decide it was "contracts" for all the endpoints?

Reply:
Done